lib/chess_web/views/game_view.ex

```diff
@@ -3,6 +3,14 @@ defmodule ChessWeb.GameView do
 
   import Chess.Auth, only: [current_user: 1]
 
+  def your_turn?(conn, game) do
+    player_colour(conn, game) == game.turn
```

Comment:
It feels strange that this function relies on a colour (man that is really hard to type as an american...) instead of using the underlying information which seems to be `current_user(conn).id == game.user_id`. If the color of pieces change, this functionality has to change and that feels like strange coupling.

Reply:
Maybe something like:

```elixir
  def your_turn?(conn, game) do
    current_user(conn, game) == game.user_id
  end

  def player_colour(conn, game) do
    your_turn(conn, game) && "white" || "black"
  end
```
to invert the relationship.